docs/news.rst

```diff
@@ -3,12 +3,15 @@ Release Notes
 
 **UNRELEASED**
 
+- Added a ``wheel tags`` command to modify tags on an existing wheel.
 - Updated vendored ``packaging`` to 23.0
 - ``wheel unpack`` now preserves the executable attribute of extracted files
 - Fixed spaces in platform names not being converted to underscores (PR by David Tucker)
 - Fixed ``RECORD`` files in generated wheels missing the regular file attribute
 - Fixed ``DeprecationWarning`` about the use of the deprecated ``pkg_resources`` API
   (PR by Thomas Grainger)
+- Wheel now uses flit-core as a build backend, removing a circular dependence
+  for setuptools.
```

Comment:
This is a bit misleading as it didn't have a declared run-time dependency on setuptools in the previous release.

Reply:
```suggestion
- Wheel now uses flit-core as a build backend.
```

Simple fine?